CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change log
 
+### 0.2.2
+- Added functionality to get the specific umber of errors
```

Comment:
its `number` . Please check typo
- Added functionality to return after a specific number of errors

Reply:
my bad